### DISCUSSION TITLE
Improving the DAM file structure

### DISCUSSION THREAD

**Discussion by smserpa** [zesty-io/manager-ui#1093] [Ideas]
The Zesty DAM stores and optimizes media files in the Zesty cloud. Here users can organize files into sub-categories and search by file name. We are constantly evolving to make user experiences better so lets start the conversation on how the DAM file structure can improve!

## We Want to Hear Your Feedback! ## 

Do you want to see certain sort or filtering options incorporated?

If so, how would you like Zesty to sort/filter files and folders?

Alphabetically? Date added?

What about presentation?

Toggle? Dropdown?

## Current UI ##

![Screen Shot 2022-05-18 at 7 29 21 PM](https://user-images.githubusercontent.com/69278400/169376761-eed879d4-e757-4c32-b4af-a3dc02a7d53d.png)

## Update Ideas ##

![Screen Shot 2022-05-19 at 2 41 40 PM](https://user-images.githubusercontent.com/69278400/169377633-1053aa4b-6741-4332-9f20-cb67e04b1dd0.png)
<br>
What about accessing files?

Do you like the On-The-Fly Image Editor options? Do you want to see more options or have any feedback?

## Current UI ##

![Screen Shot 2022-05-19 at 3 20 16 PM](https://user-images.githubusercontent.com/69278400/169386848-3166d297-9deb-461a-a6a2-ea3f37f67b72.png)

**Comment by chloespilotro** [comment]
LOVE the idea of being able to filter based on file type, that would be a great workflow improvement.

I work in Ecosystems often, so I'd also like the ability to toggle between EcoMedia and Instance Media - in Ecosystems, it can sometimes be tough to go through _all_ of the folders available to find the one I'm looking for if I'm just looking for media folders for that particular instance.

**Comment by dhakimzadeh** [comment]
few things:

- tagging and categories and collections | both on the fly and pre made / basically need a settings area for the dam
- example i like https://burton.getbynder.com/media/
- good ways to browse your assets based on collections, categories, and tags // something straight forward like collections get a menu item, categories are in the dropdowns, tags used in search
![image](https://user-images.githubusercontent.com/59213/169409321-16bb71b8-8b57-4331-b39b-03c125665a82.png)

- ability to get clean link with one click 
- option to obfuscate filenames on first upload (like they rename now if you upload two of the same thing) !!!!!!!
- all the image manipulation options and metadata, grouped in accordions like the screenshot, 
- on the crop tool accordion, the ability to drag a select box on the image and have it fill out the coordinates- stuff like this where applicable, like with pad option
- pre-set derivatives that i can configure somewhere, then they are there for all admins, also 1 click copy
- ability to insert these into WYSIWYG content with the wysiwyg add button!!!
- granular permissions for this stuff, maybe i want them uploading and tagging but not able to crop or create derivatives other than my presets
![image](https://user-images.githubusercontent.com/59213/169410094-69f29075-426f-4707-9eca-276e3d907454.png)

![image](https://user-images.githubusercontent.com/59213/169411051-f150ba45-3b13-4da7-9331-edfdb90ba0ce.png)

https://burton.getbynder.com/collections/view/F6BADF75-9E87-4734-91FCA646A3A1EC91/?mediaId=C04A1E32-2C57-4B33-B010773316EE8E09

**Reply by smserpa** [reply]
Great suggestions! I really like grouping all settings, image manipulation, and meta into accordions. Thanks for the visual examples too!

**Comment by jjnotjayjay** [comment]
- Ability to move an image from one folder to another post-upload: https://zestyiodevs.slack.com/archives/C1RRP2B0A/p1653329118213439

- Don't default to including ?width=...&height= query params, it's annoying and doesn't mirror other common CMSes.

Other than that, honestly I think the DAM is fine and that, IMO, there are more pressing things with the Zesty platform to spend time improving.

**Reply by smserpa** [reply]
Thanks @jjnotjayjay! Appreciate your feedback!

**Reply by jjnotjayjay** [reply]
Wanted to add one more request that's more for WYSIWYG editors but does involve the DAM: just like how we can restrict a Media/Images Field to a specific folder, it'd be very useful to restrict media uploads within a WYSIWYG field to a specific folder.

Clients accidentally upload images to the uncategorized DAM folder from within WYSIWYGs all the time, which coupled with the inability to move files from one folder to the next, leads to a lack of organization over time.

**Comment by snvboy** [comment]
Sorting folders by name/date is critical. Moving content from one folder to another would be very useful to fixing cases where media is poorly organized by users.

**Comment by snvboy** [comment]
Versioning of media. We frequently have to make edits/updates to content, and this involves uploading the new content as a new media object, and then going in to all the content items and updating which media object they are using. It would be much more convenient to be able to upload a new version and use the same ZUID so no changes in the content items are needed.

**Comment by zcolah** [comment]
Hello everyone,

Great ideas shared here so far! I'm Zosh, the product experience designer at Zesty and am now starting to design the new DAM experience. I'd really love to hear your feedback through our [feedback survey](https://forms.gle/qGSZ7t2Xv8wRxonh7). 

This will help us better understand which new features you'd find most useful so that we can accordingly prioritize what to build first. 

Thank you so much in advance! 

**Zosh,** 
Product Designer @ Zesty.io
